docs/languages/csharp/properties.md

```diff
@@ -0,0 +1,255 @@
+# Introduction
```

Comment:
Let's have you list your name as the author.  Want to make sure you get the credit for the awesome guide!  Should be a link to your Github profile.  e.g.

Author: [Bill Wagner](https://github.com/BillWagner)